docs/tools-offline-instructions/info/projectModules.json

```
@@ -0,0 +1,96 @@
+{
+    "@babel/eslint-parser": [
+        "7.14.7"
+    ],
+    "@sap-ux/eslint-plugin-fiori-tools": [
+        "^0.3.1"
+    ],
+    "@sap/eslint-plugin-ui5-jsdocs": [
+        "2.0.5"
+    ],
+    "@sap/ui5-builder-webide-extension": [
+        "^1.1.8"
+    ],
+    "@sap/ux-specification": [
+        "1.120.8",
+        "1.71.106",
+        "1.84.86",
+        "1.96.60",
+        "1.102.39",
+        "1.108.27"
```

@sap/ux-specification version `1.102` will be deprecated soon, should we remove it already?
```suggestion
        "1.108.27"
```

---

This file is automatically generated file, I'd like to avoid manual changes to this.